docs/index.md

```diff
@@ -1,226 +1,35 @@
 ---
 layout: index
-title: SW-Helpers
+title: Welcome to Workbox
 navigation_weight: 0
 ---
 
-<!-- DO NOT EDIT. This page is autogenerated. --><!-- To make changes, edit templates/README.hbs, not README.md! -->
-[![Travis Build Status][travis-image]][travis-url]
-[![AppVeyor Build status][appveyor-image]][appveyor-url]
-[![Dependency Status][dependency-image]][dependency-url]
-[![Dev Dependency Status][dev-dependency-image]][dev-dependency-url]
+# Welcome to Workbox
 
-# SW-Helpers
+Workbox is a suite of helper libraries for service workers and progressive web
+apps designed to simplify your development experience. 
 
-## Contents
+The easiest workbox deployments implement precaching in a manner of minutes.
+While the full set of features supports such things as runtime caching, routing,
```

Comment:
This sentence isn't grammatically correct, and the whole thing flows awkwardly.

How about:

_Using Workbox, you can implement precaching in a matter of minutes.
If you want to go further, Workbox provides routing logic, runtime caching, offline analytics, background syncing, and more."_